.github/workflows/build-docker-image.yml

```diff
@@ -107,6 +107,13 @@ jobs:
           password: ${{ steps.auth.outputs.access_token }}
           logout: false
 
+      - name: Login to DockerHub
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
If we decide to remove the tag filter, we'll want to remove it here as well.

Reply:
Fixed with [ci(release): just publish to DockerHub when a release is published](https://github.com/ZcashFoundation/zebra/pull/5138/commits/53191eaea9016ac8b0cd7633d77ed610fa9be62c)